base_view_inheritance_extension/readme/DESCRIPTION.rst

```diff
@@ -1,2 +1,3 @@
 This module was written to make it simple to add custom operators for view
 inheritance.
+You need to install the Python 'astor' library to use it.
```

Comment:
Could you write it in the dedicated install.rst file ?

Reply:
oh yes, my mistake